common/common.sh

```diff
@@ -0,0 +1,746 @@
+#!/usr/bin/env bash
+###############################################################################
+# SPDX-License-Identifier: GPL-2.0-only                                       #
+# Copyright (c) 2022 Intel Corporation.                                       #
+#                                                                             #
+# Common Bash Functions                                                       #
+###############################################################################
+
+# prevent duplicate sourcing
+[[ -n $lkvs_lib_imported ]] && exit 0
+lkvs_lib_imported=1
+
+# Check whether current user is root, if current user
+# is not root, exit directly.
+user_check() {
+  local user=""
+
+  if command -v whoami &> /dev/null; then
+    user=$(whoami)
+  elif command -v id &> /dev/null; then
+    user=$(id -nu)
+  else
+    user="${USER}"
+  fi
+
+  if [[ "${user}" != "root" ]]; then
+    echo "LKVS must run with root privilege!"
+    exit 1
+  fi
+}
+
+user_check
```

Comment:
Since we assume that all test cases require root privileges, putting the check hre will prevent the case owner from having to deal with this logic in every test case.